src/bot.py

```diff
@@ -43,138 +43,13 @@ def __init__(self, token):
-        await ctx.send(self.NON_COMMANDER_ERROR)
-        return
-
-      new_probability = int(new_probability)
-      if new_probability <= 0:
-        await ctx.send('New roll must be greater than zero.')
-      else:
-        self.probability = new_probability
-        logging.info(f'Updated probability to {new_probability}')
-        await ctx.send(f'Updated roll to d{new_probability}.')
-    except ValueError:
-      await ctx.send(f'{new_probability} is not an integer.')
-    except Exception as e:
-      logging.error(e, exc_info=True)
-
-
-  async def _command_clear_hermables(self, ctx):
-    try:
-      if not self._is_commander(ctx.author.name):
-        await ctx.send(self.NON_COMMANDER_ERROR)
-      elif len(self.hermables) == 0:
-        await ctx.send("I don't have any friends right now.")
-      else:
-        self.hermables.clear()
-        logging.info('cleared hermables')
-        await ctx.send("Friends cleared.")
-    except Exception as e:
-      logging.error(e, exc_info=True)
-
-
-  async def _command_join(self, ctx):
-    try:
-      if not self._is_commander(ctx.author.name):
-        await ctx.send(self.NON_COMMANDER_ERROR)
-      elif ctx.author.voice is None:
-        await ctx.send("You are not in a voice channel.")
-      elif self.voice_client is not None and self.voice_client.channel.name == ctx.author.voice.channel.name:
-        await ctx.send("I'm already in your voice channel.")
-      else:
-        if self.voice_client is not None:
-          await self._leave_voice_channel()
-
-        await self._join_voice_channel(ctx.author.voice.channel)
-        self.voice_client.listen(discord.reader.ConditionalFilter(self.vad_sink, self._should_herm_user))
-    except Exception as e:
-      logging.error(e, exc_info=True)
-
-
-  async def _command_leave(self, ctx):
-    try:
-      if not self._is_commander(ctx.author.name):
-        await ctx.send(self.NON_COMMANDER_ERROR)
-      elif self.voice_client is None:
-        await ctx.send("I'm not currently in a voice channel.")
-      else:
-        await self._leave_voice_channel()
-    except Exception as e:
-      logging.error(e, exc_info=True)
-
-
-  async def _command_list_hermables(self, ctx):
-    try:
-      if not self._is_commander(ctx.author.name):
-        await ctx.send(self.NON_COMMANDER_ERROR)
-      elif len(self.hermables) < 1:
-        await ctx.send("I don't have any friends right now.")
-      else:
-        friends_list = '• ' + '\n• '.join(sorted(self.hermables))
-        await ctx.send(f'My friends:\n{friends_list}')
-    except Exception as e:
-      logging.error(e, exc_info=True)
-
-
-  async def _command_print_probability(self, ctx):
-    try:
-      if not self._is_commander(ctx.author.name):
-        await ctx.send(self.NON_COMMANDER_ERROR)
-      else:
-        await ctx.channel.send(f'Roll is currently d{self.probability}')
-    except Exception as e:
-      logging.error(e, exc_info=True)
+    self.command_prefix='|'
```

Comment:
You don't need this line, when you set the prefix in `super().init()`, that essentially does the same thing.

Also, change prefix back to '~'